docs/pipelines.md

```diff
@@ -763,7 +763,7 @@ tasks:
 ### Configuring the failure timeout
 
 You can use the `Timeout` field in the `Task` spec within the `Pipeline` to set the timeout
-of the `TaskRun` that executes that `Task` within the `PipelineRun` that executes your `Pipeline.`
+of the `TaskRun` (and its `retries`) that executes that `Task` within the `PipelineRun` that executes your `Pipeline.`
```

Comment:
reading this doc and looking at the example below, it's not clear to me if `timeout: "0h1m30s" would be for each individual TaskRun (and each individual retry) or for the time taken in total across the TaskRun + Retries?

* if individual, maybe we could say something like "the time allowed for execution of an individual TaskRun, and for each of it's subsequent retries
* if it's the totaly, maybe something like "the total amount of time allowed for the execution of an individual taskRun and all of it's retries"
* 
(i'm actually not sure this is clearer .. haha this is hard... maybe we can get some input from @geriom XD)

Reply:
`Tasks` can define their own timeout, so my expectation here was that this was provided to override that at `Pipeline` level. It could be interpreted as a timeout for the sum of all retries of a `Task`, even though that feels a bit strange to me. Either case I agree we should make it crystal-clear in the docs what this timeout is about :)